test/drafts/Strings.test.js

```diff
@@ -0,0 +1,49 @@
+const { constants } = require('openzeppelin-test-helpers');
+
+const StringsMock = artifacts.require('StringsMock');
+
+contract('Strings', function () {
+  const a = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
```

Comment:
See my note below regarding using variables, but regardless I think it'd be better to use strings with distinct characters for these sort of thing, otherwise subtle errors may be covered. I usually go for stuff like `'OpenZeppelin'`, but go wild :)

Reply:
Updated tests